### ISSUE TITLE
JSX - Space inserted after `<` when a component is supplied as a value to {}

### ISSUE THREAD

**Issue by digita1-Anal0g** [Glavin001/atom-beautify#1477]
# Description

When using [composition](https://facebook.github.io/react/docs/composition-vs-inheritance.html#containment) to define components, a white space is inserted after the opening `<` tag in the declaration (e.g. `someAttr={<SomeComponent />}`)

I'm using PrettyDiff as the Beautifier for JSX file extensions.

# Input Before Beautification

This is what the code looked like before:

    class SplitPane extends React.Component {
        constructor(props) {
            super(props);
        }

        render() {
            return (
                <div className='SplitPane'>
                    <div className='SplitPane-left'>
                        {props.left}
                    </div>
                    <div className='SplitPane-right'>
                        {props.right}
                    </div>
                </div>
            );
        }
    }

    class Contacts extends React.Component {
        constructor(props) {
            super(props);
        }

        redner() {
            return (
                <div>Contacts</div>
            );
        }
    }

    class Chat extends React.Component {
        constructor(props) {
            super(props);
        }

        redner() {
            return (
                <div>Chat</div>
            );
        }
    }

    class App extends React.Component {
        constructor(props) {
            super(props);
        }

        render() {
            return (
                <SplitPane left={<Contacts />} right={<Chat />}/>
            );
        }
    }

# Expected Output

The beautified code should have looked like this:

Other code omitted for brevity

    class App extends React.Component {
        constructor(props) {
            super(props);
        }

        render() {
            return (
                <SplitPane left={<Contacts />} right={<Chat />}/>
            );
        }
    }

# Actual Output

Other code omitted for brevity

The beautified code actually looked like this:

    class App extends React.Component {
        constructor(props) {
            super(props);
        }

        render() {
            return (
                <SplitPane left={< Contacts />} right={< Chat />}/>
            );
        }
    }

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug

Here is a link to the [debug.md](https://gist.github.com/digita1-Anal0g/1e4eb709ee60ef55c1986725a7f3431a) Gist:

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [x] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by prettydiff** [comment]
I do not see the debug.md file, but I suspect this is a Pretty Diff defect.  I can reproduce the defect using the Pretty Diff that ships with Atom Beautify, but cannot reproduce this in the current version of Pretty Diff.

Last night/this morning I have resolved a blocking bug that has held me up for a month.  I am getting some kind of windows only regression and I hope to have this resolved today.  Once this minor speed bump is cleared I can look at next steps for recruiting users to test this out to ensure durability before introducing it into Atom Beautify in production.

* https://semaphoreci.com/prettydiff/biddle/branches/0-1-17/builds/6
* https://ci.appveyor.com/project/prettydiff/biddle/build/1.0.84

**Comment by prettydiff** [comment]
Keeping everybody informed I have fixed biddle.  Check it out now - https://github.com/prettydiff/biddle

I will republish Pretty Diff using biddle in production tomorrow.  I am not ready to give it to Atom Beautify just yet though.  I actually need users to play around with this thing for a bit so that I can get a sense of where the pain points are and just how bad it works for fails for other people.  I wrote a getting started document at https://github.com/prettydiff/biddle/blob/master/documentation/gettingstarted.md

The documentation might be easier to read from the command line though as I included a markdown to CLI parser in biddle.  The number at the end defines the word wrap limit:

    node biddle markdown documentation/gettingstarted.md 80
